### PR TITLE
Ch update profile #39

### PR DESCRIPTION
**Pull request by gyssa-prince**
#### What does this PR do?

A driver and operator** will be able to update their information.

#### Description of Task to be completed?

-The operator and driver should be able to update information about their own profile.
-The operator and driver profile should be able accessible publicly.
-Appropriate error messages are displayed on unsuccessful attempts
-Skeleton UIs/screen are/is shown when the page loads/is loading..
-The page is responsive.

How should this be manually tested?
-clone repo
-run npm install
-git checkout ch-UpdateProfile-#39
-npm run dev:start
Routes:
-User profile: http://localhost:3000/profile
-To change password: http://localhost:3000/profile/changepassword

Any background context you want to provide?
-N/A

What are the relevant pivotal tracker stories?
-Link:https://trello.com/c/h4b8nCl7

Screenshots (if appropriate
Screenshot 2022-05-03 213711
mobile

### CODE REVIEW

**Review by Juleshb — DISMISSED** [review]

**Review by songa1 — COMMENTED** [review]
Remove dark colors on your buttons. And add a footer.

**Review by arthurjunior250 — CHANGES_REQUESTED** [review]
your content is not in the middle according to the design

**Review by Benafrica — DISMISSED** [review]

**Review by shemalucien — DISMISSED** [review]

File: src/components/skeletonUI.js [hunk 1]
```diff
@@ -1,12 +1,9 @@
 import React from 'react'
-import loader from '../../assets/loading.gif'
+import loader from './img/loading.gif'
 export default function skeletonUI() {
   return (
     <div className="flex flex-col items-center">
-      <img src={loader} alt='' />
+      <img src={loader} alt='' className='pt-32 w-72 h-70'/>
```
Comment (arthurjunior250) [inline src/components/skeletonUI.js]: remove this className ,your loader will look good

**Review by arthurjunior250 — CHANGES_REQUESTED** [review]
check your popup notification, it is hidden

File: src/components/skeletonUI.js (hunk 1 above)
Comment (gyssa-prince) [reply src/components/skeletonUI.js]: Good

**Review by Juleshb — APPROVED** [review]

**Review by songa1 — APPROVED** [review]

**Review by arthurjunior250 — APPROVED** [review]

**Pull request merged**